InriaMuseum/InriaMuseum.css

```diff
@@ -0,0 +1,7 @@
+#button_suite_group,#button_start_fr,#button_active,#button_leave,#button_music1,#button_music2,#button_défis,#button_défis-2:hover{
```

Comment:
why all of these classes, please remove everything that is not used

Reply:
All of them are used is the id of the button to transform the mouse in an hand